### PR TITLE
Fix the link in the toc

### PR DESCRIPTION
**Pull request by queenypingcap**
If the topic with a link is in the table of content, the mark should be '-' instead of '+'.

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by queenypingcap** [comment]
@xuechunL PTAL at the change regarding the toc.

**Comment by queenypingcap** [comment]
@GregoryIan PTAL at the change regarding the loader configuration.

File: tools/loader.md [hunk 1]
```diff
@@ -104,4 +114,5 @@ Or use configuration file "config.toml":
     
 ### Note
 
-If you use the default checkpoint file, after importing the data of a database, please delete loader.checkpoint before you begin to import the next database. When importing each database, explicitly specify the file name of checkpoint is recommended.
++ If you use the default checkpoint file, after importing the data of a database, please delete loader.checkpoint before you begin to import the next database. When importing each database, explicitly specify the file name of checkpoint is recommended.
```
Comment (GregoryIan) [inline tools/loader.md]: If you use the default checkpoint-schema, after importing the data of a database, please drop database tidb_loader before you begin to import the next database

**Comment by queenypingcap** [comment]
@GregoryIan PTAL.

File: tools/loader.md [hunk 2]
```diff
@@ -114,5 +114,5 @@ Or use configuration file "config.toml":
     
 ### Note
 
-+ If you use the default checkpoint file, after importing the data of a database, please delete loader.checkpoint before you begin to import the next database. When importing each database, explicitly specify the file name of checkpoint is recommended.
++ If you use the default checkpoint-schema, after importing the data of a database, drop the tidb_loader database before you begin to import the next database. When importing each database, explicitly specify the file name of checkpoint is recommended.
```
Comment (holys) [inline tools/loader.md]: > explicitly specify the file name of checkpoint is recommended.
No longer file name but `checkpoint-schema`

**Comment by GregoryIan** [comment]
LGTM

**Comment by queenypingcap** [comment]
@holys PTAL.

**Comment by holys** [comment]
LGTM

**Pull request merged**